src/main/java/app/mapping/PointData.java

```diff
@@ -18,10 +18,10 @@ public enum PointData implements ExchangeParam {
 
 	public String toString() {
 		return switch (this) {
-			case TRENDBREAK -> "Trendlinien-Brüche";
-			case EQMOVINGAVG -> "Gleich gleitendem Preis";
-			case EQSUPPORT -> "Gleich Support-Linie";
-			case EQRESIST -> "Gleich Resist-Linie";
+			case TRENDBREAK -> "Trendbrüche";
+			case EQMOVINGAVG -> "Durchbruch - Gleitender Schnitt";
+			case EQSUPPORT -> "Durchbruch - Support Linie";
+			case EQRESIST -> "Durchbruch - Widerstandslinie";
```

Comment:
@JakobPK sind EQSUPPORT und EQRESIST wirklich Durchbrüche von Unterstützungs-/Widerstandslinien? Soweit ich dich letztes Mal verstanden hatte ist einfach nur, wenn der Preis gleich dem globalen Minimum/Maximum ist (so ist zurzeit auch implemetiert). In dem Fall würde ich "Durchbruch" ehe irreführend finden

Reply:
Hmmm, wie wäre einfach nur "Kurs trifft Support" oder "Kurs = Support", um den Namen kürzer zu halten?

Reply:
@JakobPK @ArtInLines Ich hatte `Preis = Stütz`, `Preis = Schnitt`, `Preis = Widerst.` geschrieben. Was haltet ihr davon?

Reply:
Gefällt mir